tutor/src/components/copy-on-focus-input.cjsx

```diff
@@ -0,0 +1,33 @@
+React = require 'react'
+
+omit = require 'lodash/omit'
+
+Clipboard = require '../helpers/clipboard'
+
+module.exports = React.createClass
+  displayName: 'CopyOnFocusInput'
+  propTypes:
+    value: React.PropTypes.string.isRequired
+    focusOnMount: React.PropTypes.bool
```

Comment:
since `focusOnMount` is not required, should we maybe have a defined default in `getDefaultProps`?